src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -179,10 +181,20 @@ public class Thread implements Runnable {
     @SuppressWarnings("removal")
     private AccessControlContext inheritedAccessControlContext;
 
-    /* For autonumbering anonymous threads. */
-    private static int threadInitNumber;
-    private static synchronized int nextThreadNum() {
-        return threadInitNumber++;
+    /* For autonumbering anonymous threads. It is modified from
```

Comment:
Why is this modification needed in JDK 17, but not in JDK 8 and 11？

The coroutine switching outside Wisp is allowed, and the coroutine switching on the critical path of Wisp has already been handled by the in_critical mechanism.

Reply:
After discussion, we decide to remove changes in this file.